bots/image-download

```diff
@@ -209,10 +224,31 @@ def remember_cwd():
     finally:
         os.chdir(curdir)
 
-def prune_images(force, dryrun):
+def get_image_names(quiet=False, open_pull_requests=True, offline=False):
+    """Return all image names used by all branches and optionally in open pull requests
+    """
+    images = set()
+    # iterate over visible refs (mostly branches)
+    # this hinges on being in the top level directory of the the git checkout
+    with remember_cwd():
+        os.chdir(os.path.join(BOTS, ".."))
+        refs = get_refs(open_pull_requests, offline)
+        # list images present in each branch / pull request
+        for name, ref in refs.items():
+            if not quiet:
+                sys.stderr.write("Considering images from {0} ({1})\n".format(name, ref))
+            for link in get_image_links(ref, "bots/images"):
```

Comment:
This needs to consider the old image links like https://github.com/cockpit-project/cockpit/pull/6511/files#diff-91824d32b0dcaf20aef463b2982b2b90R269